homeassistant/components/solax/manifest.json

```diff
@@ -0,0 +1,11 @@
+{
+    "domain": "solax",
+    "name": "Solax Inverter",
+    "documentation": "https://www.home-assistant.io/components/solax",
+    "requirements": [
+      "solax==0.0.3"
+    ],
+    "dependencies": [],
+    "codeowners": []
```

Comment:
I would recommend adding yourself here so that you will be tagged in bugs/PRs that affect this component.